src/pageLayout/constants/navigationHierarchy.ts

```diff
@@ -36,7 +40,8 @@ export interface NavItem {
 export const generateNavItems = (): NavItem[] => {
   const state: AppState = getStore().getState()
   const orgID = state?.resources?.orgs?.org?.id ?? ''
```

Comment:
We can't remove references to the IDPE org id from NavItem, since those are necessary to build the nav bar in OSS.